traffic_stops/middleware.py

```diff
@@ -2,7 +2,7 @@
 
 from django.utils.deprecation import MiddlewareMixin
 
-STATES = ('nc', 'md', 'il')
+STATES = ('nc')
```

Comment:
```suggestion
STATES = ('nc',)
```

I think this needs a comma to be a list